frame/recover/src/lib.rs

```diff
@@ -0,0 +1,339 @@
+// Copyright 2017-2020 Parity Technologies (UK) Ltd.
```

Comment:
I think this file is from your other open PR and should be removed from this branch.